### PR TITLE
minor fix of li52 and nsc recipes

### PR DESCRIPTION
**Pull request by sw005320**

### CODE REVIEW

**Comment by sw005320** [comment]
I'll add results and updated configs later.

File: egs2/open_li52/asr1/run.sh [hunk 1]
```diff
@@ -15,63 +15,69 @@ lid=true # whether to use language id as additional label
...
     --local_data_opts "--langs ${langs} --stage 0 --lid ${lid} --nlsyms_txt ${nlsyms_txt}" \
     --stage 1 \
     --stop_stage 100 \
-    --nj 40 \
-    --ngpu ${ngpu} \
+    --ngpu 4 \
+    --nj 80 \
     --use_lm false \
-    --token_type char \
+    --token_type bpe \
```
Comment (ftshijt) [inline egs2/open_li52/asr1/run.sh:80]: We might need to add the `nbpe` here as the default one is set to 30.

**Comment by ftshijt** [comment]
Thanks for your updates! Shall I wait for you to finish this PR or merge my updates to this PR?

File: egs2/open_li52/asr1/run.sh (hunk 1 above)
Comment (sw005320) [reply egs2/open_li52/asr1/run.sh:80]: Thanks! I'll add it.

**Comment by sw005320** [comment]
> Thanks for your updates! Shall I wait for you to finish this PR or merge my updates to this PR?

I think we can merge this PR after I fix your comments and pass CIs. Then, you can make some follow up PR.

File: egs2/open_li52/asr1/run.sh (hunk 1 above)
Comment (sw005320) [reply egs2/open_li52/asr1/run.sh:64]: Thanks. I added it. I also added some other options.

**Pull request merged**